src/scikit_build_core/setuptools/build_cmake.py

```diff
@@ -19,7 +19,7 @@
 
     from .._compat.typing import Literal
 
-__all__ = ["BuildCMake", "cmake_source_dir", "finalize_distribution_options"]
+__all__ = ["BuildCMake", "cmake_command", "cmake_extension", "cmake_source_dir"]
```

Comment:
```suggestion
__all__ = ["BuildCMake", "cmake_extension", "cmake_source_dir", "finalize_distribution_options"]
```

Reply:
Of course it would match here. I totally did not pay attention to that line in my initial patch.

Reply:
Are you OK with me using github "commit suggestion" button or do you prefer a manual process?

Reply:
Sure, I’ll squash at the end.

Reply:
Though I think we’re going to have to try slightly different approach, basically by adding it to every keyword argument, rather than adding a new keyword argument that we don’t support (yet?).